source/features/fork-source-link-same-view.tsx

```diff
@@ -8,36 +8,43 @@ import getDefaultBranch from '../github-helpers/get-default-branch';
 import {getRepo, getForkedRepo} from '../github-helpers';
 
 const isFilePath = (): boolean => pageDetect.isSingleFile()
-	|| (pageDetect.isRepoTree() && !pageDetect.isRepoRoot())
+	|| pageDetect.isRepoTree()
 	|| pageDetect.hasFileEditor();
 
-async function init(): Promise<void> {
+async function getEquivalentURL(): Promise<string> {
 	const forkedRepository = getRepo(getForkedRepo())!;
+	const defaultUrl = '/' + forkedRepository.nameWithOwner;
+
+	if (pageDetect.isConversation() || pageDetect.isRepoRoot()) {
```

Comment:
Sorry for the late review, but better late than never 🙂

I use `isIssue() || isPR()` because `isConversation()` does not include other tabs of PR.